test_resources/csv/utf-8.csv

```diff
@@ -0,0 +1,3 @@
+"Dirección";"País"
+"foo";"bar"
+
```

Comment:
I was surprised these files weren't in the PR earlier but assumed they must have existed from before (or CI would fail). I take it CI failed, haha